data/sql/V1__create.sql

```diff
@@ -1,35 +1,103 @@
--- MySQL Script generated by MySQL Workbench
--- mar 11 gen 2022, 15:53:23
--- Model: New Model    Version: 1.0
--- MySQL Workbench Forward Engineering
-
-SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0;
-SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0;
-SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='ONLY_FULL_GROUP_BY,STRICT_TRANS_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_ENGINE_SUBSTITUTION';
-
--- -----------------------------------------------------
--- Schema divetimp
--- -----------------------------------------------------
-
--- -----------------------------------------------------
--- Schema divetimp
--- -----------------------------------------------------
-CREATE SCHEMA IF NOT EXISTS `divetimp` ;
-USE `divetimp` ;
-
--- -----------------------------------------------------
--- Table `divetimp`.`Tasks`
--- -----------------------------------------------------
-DROP TABLE IF EXISTS `divetimp`.`Tasks` ;
-
-CREATE TABLE IF NOT EXISTS `divetimp`.`Tasks` (
-  `t_id` INT NOT NULL,
-  `t_name` VARCHAR(255) NOT NULL,
-  PRIMARY KEY (`t_id`))
-ENGINE = InnoDB
-DEFAULT CHARACTER SET = utf8;
-
-
-SET SQL_MODE=@OLD_SQL_MODE;
-SET FOREIGN_KEY_CHECKS=@OLD_FOREIGN_KEY_CHECKS;
-SET UNIQUE_CHECKS=@OLD_UNIQUE_CHECKS;
+-- phpMyAdmin SQL Dump
+-- version 4.9.1
+-- https://www.phpmyadmin.net/
+--
+-- Host: localhost
+-- Generation Time: Jan 13, 2022 at 08:54 PM
+-- Server version: 8.0.18
+-- PHP Version: 7.3.11
+
+SET SQL_MODE = "NO_AUTO_VALUE_ON_ZERO";
+SET AUTOCOMMIT = 0;
+START TRANSACTION;
+SET time_zone = "+00:00";
+
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!40101 SET NAMES utf8mb4 */;
+
+--
+-- Database: `divetimp`
+--
+CREATE DATABASE IF NOT EXISTS `divetimp` DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci;
```

Comment:
Apparently in the future `database` is actually a `schema` so it aligns more with other DB systems such as PostgreSQL.